suzieq/engines/pandas/engineobj.py

```diff
@@ -276,6 +275,9 @@ def get(self, **kwargs) -> pd.DataFrame:
         if not self.iobj.table:
             raise NotImplementedError
 
+        # Make sure we fetch all columns necessary including those that are
+        # needed to satisfy the filters
+
```

Comment:
Is this a TODO?

Reply:
No, just a comment.

Reply:
Ok, but I don't see the code implementing that comment. Am I missing something?

Reply:
Dead comment :) Removed.